src/jobs/models/aws_fargate/jobConnector.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const AWS = require('aws-sdk');
+
+module.exports.runJob = async (jobFargateTemplate, job) => {
+    const ecs = new AWS.ECS({ region: job.tag });
+    await ecs.runTask(jobFargateTemplate).promise();
+};
+module.exports.stopRun = async (jobPlatformName, job) => {
+    const ecs = new AWS.ECS({ region: job.tag });
+    let runningTasks = await ecs.listTasks({ desiredStatus: 'RUNNING' }).promise();
+    let describeTaskPromise;
```

Comment:
```suggestion
    let describeTaskArray;
```

since it's array of values (all promises are resolved)